grails-app/domain/com/careerfly/user/User.groovy

```diff
@@ -0,0 +1,21 @@
+package com.careerfly.user
+
+class User {
+    String firstName
+    String lastName
+    String email
+    String password
+    Gender gender
+    Date dateCreated
+    Date lastUpdated
+
+    static constraints = {
+        firstName(blank: false, matches: "[a-zA-Z]+")
+        lastName(blank: false, matches: "[a-zA-Z]+")
+        email(email: true,unique: true)
```

Comment:
There should be a space before `unique`. @akshitagangrade @ankit18singh @apoorvsrivastava1116 @Smita007 @atul12594 @shubhamj15 @Sagars09